application/src/test/java/com/yahoo/application/ApplicationTest.java

```diff
@@ -67,6 +63,20 @@ public void container_and_referenced_content() {
         }
     }
 
+    /** Tests that an application with query profile sets up the QueryProfileRegistry */
+    @Test
+    public void container_and_query_profile() {
```

Comment:
Instead of comment + test name, I suggest to be as specific as possible in the test name and state it as a requirement, e.g. `application_with_query_profile_sets_up_query_profile_registry.